.moai/specs/SPEC-UPDATE-DOC-DRIFT-001/acceptance.md

```
@@ -424,487 +425,364 @@ ls internal/template/templates/.moai/config/config.yaml ; echo "template exit=$?
 ```
 
-Expected: all three of §2.2's false claims are contradicted by the tree — `loadGateSection` is
-declared and called; both `gate.yaml` files exist; the compiled default is `Enabled: true`.
+Expected: both `ls` fail; the count is `>= 3`; and every printed site is a *read* of a path inside a
+user project — migration source, v2 fingerprint detection, residue cleanup — with no write into a
+template tree.
 
-Baseline (verbatim, HEAD `d5336214e`):
+Baseline (verbatim, `7f61332ef`):
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)